tinkerbell/smee/templates/deployment.yaml

```diff
@@ -108,9 +108,17 @@ spec:
       {{- if .Values.hostNetwork }}
       hostNetwork: true
       {{- end }}
-      {{- if .Values.singleNodeClusterConfig.controlPlaneTolerationsEnabled }}
+      {{- if or .Values.deployment.tolerations .Values.singleNodeClusterConfig.controlPlaneTolerationsEnabled }}
```

Comment:
I would be OK with replacing the `controlPlaneTolerationsEnabeld` entirely in favor of complete customization.

Cc @jacobweinstock